src/ingest_validation_tools/plugin_validator.py

```diff
@@ -130,6 +131,7 @@ def validation_error_iter(base_dir: PathOrStr, assay_type: str,
         for name, obj in inspect.getmembers(mod):
             if inspect.isclass(obj) and obj != Validator and issubclass(obj, Validator):
                 sort_me.append((obj.cost, obj.description, obj))
+    sys.path.pop(0)
```

Comment:
This worries me a little... imaging that the insert on 119 is removed at some point, but we forget that this is paired. Prefer 
- `with` context, 
- or maybe just check if `plugin_dir` is already in the list, and only add if not?

Reply:
That's a fair concern.  Thank you; I found an appropriate idiom and will update the PR.